go/kbfs/libkbfs/prefetcher.go

```diff
@@ -106,6 +107,9 @@ type blockPrefetcher struct {
 	doneCh chan struct{}
 	// map to store prefetch metadata
 	prefetches map[kbfsblock.ID]*prefetch
+	// map to channels for cancelling queued prefetches
+	queuedPrefetchHandles     map[kbfsblock.ID]chan<- struct{}
```

Comment:
The usual rule we try to follow is that things protected by a specific mutex should be in its own little section, with the mutex listed first.  So these definitions should be down below, surrounded by blank lines, with the mutex first.